SU-SWS/react_paragraphs
# D8CORE-1420 Fix the ckeditor toolbar when scrolling down the item.

# READY FOR REVIEW

# Summary
- apparently `sticky` doesn't work when in the modal for whatever reasons. so the only option was to use javascript with scroll listeners to fix the position at the top of the modal.

# Need Review By (Date)
- 12/3

# Urgency
- medium

# Steps to Test
1. checkout this branch and clear browser cache
1. Create a page with a paragraph type that has a wysiwyg... cards are a good one to test.
1. scroll down the modal and see the ckeditor toolbar sticks to the top of the modal but not if you keep scrolling all the way down or back up.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
